dsl_parser/scan.py

```diff
@@ -81,40 +89,44 @@ def _scan_operations(operations,
                      context=None,
                      path='',
                      replace=False):
-    for name, definition in operations.iteritems():
-        if isinstance(definition, dict) and 'inputs' in definition:
+    for name, definition in operations.items():
+        if isinstance(definition, dict) and INPUTS in definition:
             context = context.copy() if context else {}
             context['operation'] = definition
-            scan_properties(definition['inputs'],
+            scan_properties(definition[INPUTS],
                             handler,
                             scope=scope,
                             context=context,
-                            path='{0}.{1}.inputs'.format(path, name),
+                            path='{0}.{1}.{2}'.format(path, name, INPUTS),
                             replace=replace)
 
 
 def scan_node_operation_properties(node_template, handler, replace=False):
-    _scan_operations(node_template['operations'],
+    _scan_operations(node_template[OPERATIONS],
                      handler,
                      scope=NODE_TEMPLATE_SCOPE,
                      context=node_template,
-                     path='{0}.operations'.format(node_template['name']),
+                     path='{0}.{1}.{2}'.format(NODES,
+                                               node_template['name'],
+                                               OPERATIONS),
                      replace=replace)
     for r in node_template.get('relationships', []):
         context = {'node_template': node_template, 'relationship': r}
         _scan_operations(r.get('source_operations', {}),
                          handler,
                          scope=NODE_TEMPLATE_RELATIONSHIP_SCOPE,
                          context=context,
-                         path='{0}.{1}'.format(node_template['name'],
-                                               r['type']),
+                         path='{0}.{1}.{2}'.format(NODES,
```

Comment:
tbh those changes i'd prefer to see as a separate PR, because i don't think this is related to inter-deployment dependencies